File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Control Flow Structures/Python/Excercises/program-1.py
# 1-Escribir un programa que pida al usuario un número entero y muestre por pantalla si es múltiplo de 41.


userInput = int(input('Ingrese un numero entero: '))

resultado = userInput % 41

if resultado == 0:
    print(True, '<--- Si es m ultiplo de 41')
else:
    print(False, '<--- No es multiplo de 41')

File: Getting-Practice-with-Python-and-Javascript/2- Control Flow Structures and Data Structures/Control Flow Structures/Javascript/Excercises/program-1.js
// 1-Escribir un programa que pida al usuario un número entero y muestre por pantalla si es múltiplo de 41.


// Resolution

let userInput = parseInt(prompt('Ingresa un numero entero: '));

let resultado = userInput / 41

if (resultado == 0) {
    console.log(`${true}, <--- es multiplo de 41`);
} else {
    console.log(`${false}, <--- no es multiplo de 41`);
}


// True y False en Javscript tiene que ser en minuscula true y false...